data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/S3Source.java

```diff
@@ -57,4 +60,14 @@ public void start(Buffer<Record<Event>> buffer) {
     public void stop() {
         sqsService.stop();
     }
+
+    @Override
```

Comment:
I think perhaps we should not yet include this. It helps to see it, but until the S3 source actually needs it we might want to keep it out.